src/open_inwoner/cms/cases/views/status.py

```diff
@@ -1029,3 +1029,47 @@ def get_context_data(self, **kwargs):
             "cases:case_detail_contact_form", kwargs=self.kwargs
         )
         return context
+
+
+class LegacyCaseDetailHandler(View):
+    """Redirect the legacy case detail to the current version with ZGW API group ref."""
+
+    def get(
+        self,
+        request,
```

Comment:
```suggestion
        request: Request,
```
check that we have `from requests import Request` at the top